src/main/java/tassist/address/logic/commands/RepoCommand.java

```diff
@@ -74,6 +79,8 @@ public RepoCommand(StudentId studentId, String username, String repositoryName,
     }
 
     /**
+     * Constructs a {@code RepoCommand} that assigns a repository URL to a student identified by their index.
```

Comment:
possible to write this for clarity?
```suggestion
     * Constructs a {@code RepoCommand} that assigns a repository URL to a student identified by their index in the displayed list
```